src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.Client.fs

```diff
@@ -85,7 +85,7 @@ module Client =
         if Directory.Exists dir then
             let analyzerAssemblies =
                 Directory.GetFiles(dir, "*Analyzer*.dll", SearchOption.AllDirectories)
-                |> Array.filter (fun a -> not (a.EndsWith("FSharp.Analyzers.SDK.dll")))
+                |> Array.filter (fun a -> not (a.EndsWith("FSharp.Analyzers.SDK.dll") || a.EndsWith("Test.dll")))
```

Comment:
We probably want to Regex this check a bit.
If people decide to use `.*Tests.dll` instead.